qiskit/visualization/circuit_visualization.py

```diff
@@ -448,7 +450,14 @@ def _latex_circuit_drawer(
                     if filename.endswith(".pdf"):
                         os.rename(base + ".pdf", filename)
                     else:
-                        image.save(filename, "PNG")
+                        try:
+                            image.save(filename)
+                        except ValueError:
+                            logger.warning(
+                                "WARNING This image format is not supported "
+                                "saving the image in PNG format without changing file extension."
+                            )
+                            image.save(filename, "PNG")
```

Comment:
I think the consensus was just to raise a warning/error and not convert the format. You could check that out in this [comment](https://github.com/Qiskit/qiskit-terra/issues/6448#issuecomment-846436279) . And I guess the same goes to text drawer's output as well.

Reply:
Got it. I was just a bit confused as to whether allow saving the file at all or not. I'll make the necessary changes.